### DISCUSSION TITLE
C-style comments when adding at closebrace?

### DISCUSSION THREAD

**Discussion by byron-hawkins** [uncrustify/uncrustify#3987] [Ideas]
is there a way to configure uncrustify to add c-style comments at the closebrace instead of cpp-style? In my project the convention is to make temporary comments in cpp-style (e.g., `TODO`, `FIXME`), and to make any permanent comment in c-style. So the automatically added closebrace comments create an impression like, "hey, what's this doing here?" or, "hey, don't forget about this!" So I really need these to be c-style comments. 

Someone already suggested a template for added comments, so it seems like this feature doesn't exist. Is it worth a feature request, or should I just mod my local uncrustify? I'm not sure if this kind of thing has general appeal.

**Comment by guy-maurel** [comment]
yes

**Comment by guy-maurel** [comment]
PR #4001 is a proposal.
Please test it and report.
